<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="9" hostname="jayesh-pc" failures="1" tests="3" name="Group Test" time="0.21" errors="0" timestamp="2022-04-13T12:12:03 IST">
  <testcase classname="TestCases.GroupTest" name="method1" time="0.028"/>
  <testcase classname="TestCases.GroupTest" name="method2" time="0.004"/>
  <testcase classname="TestCases.HardAssertionTest" name="method11" time="0.014">
    <failure type="java.lang.AssertionError" message="If doesn&amp;apos;t match = failed method1 expected [121] but found [12]">
      <![CDATA[java.lang.AssertionError: If doesn't match = failed method1 expected [121] but found [12]
at TestCases.HardAssertionTest.method11(HardAssertionTest.java:19)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- method11 -->
  <testcase classname="TestCases.GroupTest" name="method5" time="0.0">
    <ignored/>
  </testcase> <!-- method5 -->
  <testcase classname="TestCases.GroupTest" name="method6" time="0.0">
    <ignored/>
  </testcase> <!-- method6 -->
  <testcase classname="TestCases.HardAssertionTest" name="method5" time="0.0">
    <ignored/>
  </testcase> <!-- method5 -->
  <testcase classname="TestCases.HardAssertionTest" name="method6" time="0.0">
    <ignored/>
  </testcase> <!-- method6 -->
  <testcase classname="TestCases.GroupTest" name="method3" time="0.0">
    <ignored/>
  </testcase> <!-- method3 -->
  <testcase classname="TestCases.HardAssertionTest" name="method3" time="0.0">
    <ignored/>
  </testcase> <!-- method3 -->
  <testcase classname="TestCases.HardAssertionTest" name="method4" time="0.0">
    <ignored/>
  </testcase> <!-- method4 -->
  <testcase classname="TestCases.GroupTest" name="method4" time="0.0">
    <ignored/>
  </testcase> <!-- method4 -->
  <testcase classname="TestCases.HardAssertionTest" name="method2" time="0.0">
    <ignored/>
  </testcase> <!-- method2 -->
</testsuite> <!-- Group Test -->
